docs/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Bump `ORM_MAX_OPEN_CONNS` default from 10 to 20
+- Bump `ORM_MAX_IDLE_CONNS` default from 5 to 10
+
+Each Chainlink node will now use a maximum of 23 database connections (up from previous max of 13). Make sure your postgres database is tuned accordingly, especially if you are running multiple Chainlink nodes on a single database. If you find yourself hitting connection limits, you can consider reducing `ORM_MAX_OPEN_CONNS` but this may result in degraded performance.
```

Comment:
```suggestion
Each Chainlink node will now use a maximum of 28 database connections (up from previous max of 13). Make sure your postgres database is tuned accordingly, especially if you are running multiple Chainlink nodes on a single database. If you find yourself hitting connection limits, you can consider reducing `ORM_MAX_OPEN_CONNS` but this may result in degraded performance.
```
20 + 10 - 2, right?

Reply:
Not sure I follow. It's quite possible I've made a mistake. I was calculating as such:

    - Total max connections should now be:

    20 // ORM_MAX_OPEN_CONNS
    + 1 // postgres.EventBroadcaster
    + 1 // postgres.AdvisoryLocker
    + 1 // orm.PostgresLockingStrategy

    = 23